staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -58,6 +59,7 @@ func NewAttributesRecord(object runtime.Object, oldObject runtime.Object, kind s
 		dryRun:      dryRun,
 		object:      object,
 		oldObject:   oldObject,
+		labels:      labels,
```

Comment:
why do we need labels globally throughout the code base and not only in the webhooks. This looks wrong. Please move that logic into the webhook.